pom.xml

```diff
@@ -748,6 +762,7 @@
                                         <exclude>hive-exec-log4j2.properties</exclude>
                                         <exclude>tez-container-log4j2.properties</exclude>
                                         <exclude>org/apache/hadoop/hive/ql/io/CodecPool*.class</exclude>
+                                        <exclude>org/apache/hadoop/hive/ql/io/avro/AvroGenericRecordReader*.class</exclude>
```

Comment:
it should be `AvroGenericRecordReader.class`  + `AvroGenericRecordReader$*.class`
(same for CodecPool above)

Reply:
(nit; optional)

Reply:
technically yes. I followed the common pattern. I will send out separate PR to clean up the entries.